documentation.md

```diff
@@ -1,22 +1,83 @@
 # Mozilla Traffic Cop Documentation
 
-Traffic Cop handles redirecting users to different A/B/x variations through a query parameter.
+Traffic Cop places visitors into A/B/x cohorts, and either performs a redirect or executes a developer-specified JavaScript function.
 
 
 ## How it works
 
-After verifying the supplied configuration, Traffic Cop first makes sure the user is not currently viewing a variation (to avoid a redirect loop). It then chooses a random percentage (1-100).
+After verifying the supplied configuration, Traffic Cop chooses a variation for the visitor in one of two ways:
 
-If the percentage falls within the total percentage specified by the config, it will choose the appropriate variation and redirect the user.
+1. Checks the visitor's cookies to see if they were previously given a variation. If so, that same variation is used.
+2. If no previous variation exists, Traffic Cop generates a random number to choose a variation. If the supplied variations do not target 100% of visitors, the `novariation` value may be chosen. The chosen variation is written to a cookie for subsequent visits.
 
-If the percentage exceeds that specified by the config, no redirect will occur - i.e. the user is not chosen to participate in the experiment.
 
-If a variation is chosen, a cookie is set that will send the user back to the same variation if/when the page is again visited. (See below for setting cookie duration.)
+## Type A: Callback
+
+If the instance of Traffic Cop is provided a `customCallback` in the config, the function supplied will be passed the variation value and executed.
+
+Any further functionality is handled by the `customCallback`.
+
+**For performance, and to not require any kind of `document.ready` code, it is recommended that "Type A" instances of Traffic Cop have their `.js` files placed at the bottom of the page.**
```

Comment:
Suggest something slightly more specific like "at the bottom of the page just before the closing `</body>` tag"?